firstvoices/backend/models/signals/mtd_signals.py

```diff
@@ -20,22 +21,24 @@ def rebuild_mtd_index(site_slug):
 @receiver(pre_save, sender=DictionaryEntry)
 def store_current_visibility(sender, instance, **kwargs):
```

Comment:
I paused this signal in addition to `request_update_mtd_index` as it would also be continuously triggered by a bulk visibility change